src/utils/ui-utils.js

```diff
@@ -21,16 +21,33 @@ var UIUtils = function ($) { // jshint ignore:line
             };
         };
 
+        /**
+         * Prevent text selection
+         * With cursor drag
+         **/
+        var pauseEvent = function (e) {
+            e.stopPropagation();
+            e.preventDefault();
+            e.cancelBubble = true;
+            e.returnValue = false;
+            return false;
+        };
+
         $(element).on('mousedown', function (e) {
+            pauseEvent(e);
             var coords = getCoords(element);
             var shiftX = e.pageX - coords.left;
             var shiftY = e.pageY - coords.top;
 
             document.body.appendChild(element);
 
             var moveAt = function (e) {
-                element.style.left = e.pageX - shiftX + 'px';
-                element.style.top = e.pageY - shiftY + 'px';
+                var transform = 'translate3d(' + (e.pageX - shiftX) + 'px,' + (e.pageY - shiftY) + 'px, 0px)';
+                element.style.webkitTransform = transform;
+                element.style.mozTransform = transform;
+                element.style.msTransform = transform;
+                element.style.oTransform = transform;
+                element.style.transform = transform;
```

Comment:
It had better move this code to one method in the `uiUtils`, like `moveElementTo: function(el, x, y){...}`